builtins/amp-ad.js

```diff
@@ -150,20 +142,13 @@ export function installAd(win) {
      */
     onLayoutMeasure() {
       this.isInFixedContainer_ = this.isPositionFixed();
-      // We remeasured this tag, lets also remeasure the iframe. Should be
-      // free now and it might have changed.
-      this.measureIframeLayoutBox_();
     }
 
     /**
-     * Measure the layout box of the iframe if we rendered it already.
-     * @private
+     * @override
      */
-    measureIframeLayoutBox_() {
-      if (this.iframe_) {
-        this.iframeLayoutBox_ =
-            this.getViewport().getLayoutRect(this.iframe_);
-      }
+    getInsersectionElementLayoutBox() {
+      return this.getViewport().getLayoutRect(this.iframe_);
```

Comment:
The property that this measurement is only performed during `onLayoutMeasure` should be maintained.